### DISCUSSION TITLE
Checksum available for image files?

### DISCUSSION THREAD

**Discussion by MulberryPi** [rern/rAudio#1759] [General]
Hi all,

Are the checksums available to verify the rAudio image file(s) downloaded correctly? 

I have a quota on data allowance for my internet connection and with the image files being almost 1GB each, need a way to verify that the download completed correctly.

Thanks!

**Comment by MulberryPi** [comment]
Hi all,

Does anyone have any input into this? How can one verify their downloaded image file has integrity?

For some reason I seem to have interrupted downloads when i try get the image files from github, is there a mirror one can get these files from instead?

I am not in the USA and wonder if this is the reason I have constant issues downloading large files from github

Thanks!

**Comment by klassikmann** [comment]
@MulberryPi 
Have a look at the recent image file:
https://github.com/rern/rAudio/releases/tag/i20240818
You find checksums (MD5) and a mirror location.

**Reply by MulberryPi** [reply]
Thank you! I could've sworn i never saw those on the downloads page before, thanks to all involved if newly implemented!
